partiql-eval/src/main/kotlin/org/partiql/eval/internal/operator/rex/ExprStructStrict.kt

```diff
@@ -13,7 +12,7 @@ internal class ExprStructStrict(private val fields: List<ExprStructField>) :
         val fields = fields.mapNotNull {
             val key = it.key.eval(env)
             if (key.isNull || key.isMissing) {
-                throw TypeCheckException("Struct key was absent.")
+                throw NullPointerException("Struct key was null or missing.")
```

Comment:
should this be some other exception rather than Java's `NullPointerException`?

Reply:
See my latest commit. LMK if you're aligned.

Reply:
Returning null on null key and missing on missing key seems reasonable. Tbh not sure if the spec defines this behavior